dotlottie-ffi/emscripten_bindings.cpp

```diff
@@ -155,7 +155,8 @@ EMSCRIPTEN_BINDINGS(DotLottiePlayer)
         .function("load_state_machine", &DotLottiePlayer::load_state_machine)
         .function("start_state_machine", &DotLottiePlayer::start_state_machine)
         .function("end_state_machine", &DotLottiePlayer::end_state_machine)
-        .function("post_event_payload", &DotLottiePlayer::post_event_payload);
+        .function("post_event_payload", &DotLottiePlayer::post_event_payload)
+        .function("state_machine_framework_setup", &DotLottiePlayer::state_machine_framework_setup);
```

Comment:
Nice, I'm able to return all the Listeners, but i'm not sure if we're going to use snake_case in naming the props and values in our state machine schema in the future or we're going to mix between snake_case and camel_case, this is something that we have to decide before we ship the first schema version, what do you think ? 

<img width="562" alt="image" src="https://github.com/LottieFiles/dotlottie-rs/assets/39750790/af16fc5b-ddb9-428f-9491-d49c23f568ca">

Reply:
For the moment everything thats a String defined "Type" is using camel case I think thats the standard for all of our frameworks where this is going to be necessary

The only problem is inside dotlottie-js that it clashes because you define properties in snake_case and their values in camelCase :/

Reply:
Can decide on this and implement in a different PR